creator-node/src/services/sync/secondarySyncFromPrimary.ts

```diff
@@ -762,13 +787,11 @@ async function _secondarySyncFromPrimary({
   return { result }
 }
 
-const secondarySyncFromPrimary = instrumentTracing({
+export const secondarySyncFromPrimary = instrumentTracing({
   fn: _secondarySyncFromPrimary,
   options: {
     attributes: {
       [tracing.CODE_FILEPATH]: __filename
     }
   }
 })
-
-module.exports = secondarySyncFromPrimary
```

Comment:
I changed how the function is exported so its no longer a default export because it seems to cause a lot of confusion when switching back and force between js and ts files (no to mention it can be a headache with test files)

Reply:
👍